From 6bd98dd0cbc8cf2f398684b01d6cef4e3fbeaef2 Mon Sep 17 00:00:00 2001
From: Ioana Radulescu <ruxandra.radulescu@freescale.com>
Date: Thu, 16 Aug 2012 21:39:39 +0000
Subject: [PATCH 057/227] dts/fsl: Increase range of dynamic QMan fqids

The DPAA Ethernet driver uses dynamic fqids for the default and
error Rx/Tx queues. Since the number of Tx queues was increased to
a maximum of 24 on T4 platform, not enough fqids remain available
for all ports on T4, leading to failed probing of the last few ones.

Double the size of the pool from which dynamic fqids are allocated
to solve this problem.

Signed-off-by: Ioana Radulescu <ruxandra.radulescu@freescale.com>
[Kevin: Original patch taken from fsl sdk 1.3.1
QorIQ-SDK-V1.3.1-SOURCE-20121220-yocto.iso.]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/powerpc/boot/dts/fsl/qoriq-dpaa-res3.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/powerpc/boot/dts/fsl/qoriq-dpaa-res3.dtsi b/arch/powerpc/boot/dts/fsl/qoriq-dpaa-res3.dtsi
index 17eed8f..6574aa8 100644
--- a/arch/powerpc/boot/dts/fsl/qoriq-dpaa-res3.dtsi
+++ b/arch/powerpc/boot/dts/fsl/qoriq-dpaa-res3.dtsi
@@ -67,7 +67,7 @@
 &qportals {
 	qman-fqids@0 {
 		compatible = "fsl,fqid-range";
-		fsl,fqid-range = <256 256>;
+		fsl,fqid-range = <256 512>;
 	};
 	qman-pools@0 {
 		compatible = "fsl,pool-channel-range";
-- 
1.7.9.7

